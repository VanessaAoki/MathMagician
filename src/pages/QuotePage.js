import React from 'react';

const QuotePage = () => (
  <div className="quoteContainer">
    <p>Pure mathematics is, in its way, the poetry of logical ideas.</p>
    <p><b>â€” Albert Einstein</b></p>
  </div>
);

export default QuotePage;
