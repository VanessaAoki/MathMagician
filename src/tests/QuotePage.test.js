import React from 'react';
import renderer from 'react-test-renderer';
import QuotePage from '../pages/QuotePage';

it('Home renders as expected', () => {
  const tree = renderer.create(<QuotePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
