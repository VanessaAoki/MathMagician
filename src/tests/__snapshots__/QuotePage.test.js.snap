// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders as expected 1`] = `
<div
  className="quoteContainer"
>
  <p>
    Pure mathematics is, in its way, the poetry of logical ideas.
  </p>
  <p>
    <b>
      â€” Albert Einstein
    </b>
  </p>
</div>
`;
